ceph-pr-docs/build/build

```diff
@@ -12,4 +12,34 @@ sudo rm -f /etc/apt/sources.list.d/ceph-boost*
 sudo apt-get update -o Acquire::Languages=none -o Acquire::Translation=none
 sudo apt-get install -y gcc python3-dev python3-pip python3-virtualenv libxml2-dev libxslt-dev doxygen graphviz ant ditaa cython3
 
-timeout 3600 ./admin/build-doc
+virtualenv -q --python python3 venv
+. venv/bin/activate
+pip install tox
+pip install git+https://github.com/ceph/githubcheck.git
+sha1=$(git rev-parse refs/remotes/origin/pr/${ghprbPullId}/head)
```

Comment:
Since this is the second time I've seen this block, may I suggest it go in `build_utils.sh` instead?

Reply:
@djgalloway can we do this in another PR? after a second thought, i think we need to fix https://github.com/ceph/ceph-build/blob/01968d02ab54b07eae57c357362d0b8a2e44e730/scripts/build_utils.sh#L5-L6 first. it *always* creates a temp directory every time we include this script. and it is never removed.